save-orchestrator/src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 orchestrator.backendUrl=http://backend:5000
+orchestrator.agentLogsRelativePath=cnb/agentLogs
```

Comment:
`user.home` will be `/home/cnb`, so full path will be `/home/cnb/cnb/agentLogs`?

Reply:
Now, it will `/home/{username}/cnb/agentLogs`

Reply:
Yes, and in our normal environment, inside docker, username will be cnb